scripts/build/docker/Dockerfile.template

```diff
@@ -44,10 +44,18 @@ RUN apt-get update \
     rpm
 
 <% if (architecture != 'armv7hf') { %>
+
+# Workaround: Install a newer version of `tar` to make fpm in electron-builder work again
+RUN echo "deb http://ftp.debian.org/debian wheezy main" >> /etc/apt/sources.list
+RUN echo "deb http://ftp.debian.org/debian wheezy-backports main" >> /etc/apt/sources.list
+RUN apt-get install --force-yes -y -t wheezy libacl1 && \
```

Comment:
Ah, you need an `apt-get update` between modify `sources.list` and running `apt-get install`

Reply:
...and perhaps this should be done in a single-step `RUN` command (just like the compiler-install step below), rather than as separate `RUN` commands?

Reply:
Fixed

Reply:
I was thinking more:
```
RUN echo -e "deb http://ftp.debian.org/debian wheezy main\ndeb http://ftp.debian.org/debian wheezy-backports main" >> /etc/apt/sources.list && \
  apt-get update && apt-get install --force-yes -y -t wheezy libacl1 && \
  apt-get install --force-yes -y -t wheezy-backports tar
```
But it probably makes little difference here.